### DISCUSSION TITLE
UI issue in iPhone 7

### DISCUSSION THREAD

**Discussion by riyajath-ahamed** [community/community#63421] [Mobile] [answered]
![image](https://github.com/community/community/assets/64283797/cbd0662b-73b0-4e2f-b82d-3944cff3d37b)

- The above UI issue is cause when accessing the account with a large a bio and github badge section is overlapping.
- The bug collapse with the alignment of the profile view.
- After refreshing the bug remain as same.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.124.0 (105848740)
Device: iPhone 7 (iOS 15.7.3)
</details>

**Comment by eliperkins** [comment, marked as answer]
Hey @riyajath-ahamed, thanks for this bug report!

This bug will be fixed in an upcoming version of GitHub for iOS. I'll keep you posted when the bug fix goes live.

Thanks for helping us keep the quality of GitHub Mobile high!

**Comment by fabiocaccamo** [comment]
I confirm I have the same issue on iPhone 7 and iOS 15, I hope it will be fixed soon.

![image](https://github.com/community/community/assets/1035294/2f169565-09ce-4438-a921-b517fa9c8537)
